src/sentry/api/endpoints/user_subscriptions.py

```diff
@@ -24,18 +24,18 @@ def get(self, request, user):
         # Returns `None` if no subscriptions for user
         sub = newsletter.get_subscriptions(user)
 
-        if sub is None or not newsletter.is_enabled:
+        if sub is None or not newsletter.is_enabled():
```

Comment:
I got confused and thought this was a property instead of a method because of this:

https://github.com/getsentry/sentry/blob/master/src/sentry/web/frontend/accounts.py#L312